trappy/ftrace.py

```diff
@@ -76,6 +108,22 @@ def __init__(self, name="", normalize_time=True, scope="all",
             setattr(self, attr, trace_class)
             self.trace_classes.append(trace_class)
 
+        cache_path = _trace_cache_path(self.trace_path)
+        cache_good = False
+
+        if _check_trace_cache(self.trace_path):
+            cache_good = True
+            try:
+                # Read csv into frames
+                for trace_class in self.trace_classes:
+                    csv_file = cache_path + '/' + trace_class.unique_word[:-1] + '.csv'
```

Comment:
Yeah I don't think it's too much of a problem if the cache gets "invalidated" (especially since the cache parsing could theoretically change anyway). That actually got me thinking maybe we should also add a ".cache_version" in th cache directory. Then we could write "0" into that file and check it matches when reading back, so that if we decide to change the layout/format then we can just change that 0 and no need to worry about compatibility? Worth the effort?